### PR TITLE
fix fchk testing, v2rdm4 for L2, and test comments

### PR DESCRIPTION
**Pull request by loriab**
## Description
as it says on the tin

## Checklist
- [x] Tests added for any new features
- [x] [All or relevant fraction of full tests run](http://psicode.org/psi4manual/master/build_planning.html#how-to-run-a-subset-of-tests)

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

**Review by dgasmith — APPROVED** [review]

File: doc/sphinxman/document_tests.pl [hunk 1]
```diff
@@ -100,6 +103,9 @@
     next if $File =~ /^python$/;
     next if $File =~ /^json$/;
     next if $File =~ /^psi4numpy$/;
+    next if $File =~ /^adcc$/;
+    next if $File =~ /^brianqc$/;
+    next if $File =~ /^cppe$/;
```
Comment (andysim) [inline doc/sphinxman/document_tests.pl:108]: It's been a while since I've read perl!

File: tests/v2rdm_casscf/v2rdm4/input.dat [hunk 2]
```diff
@@ -40,6 +40,7 @@ set {
   basis cc-pvdz
   scf_type df
   d_convergence 1e-8
+  ints_tolerance 0.0
```
Comment (andysim) [inline tests/v2rdm_casscf/v2rdm4/input.dat:43]: :(

**Review by andysim — APPROVED** [review]
LGTM!

File: tests/v2rdm_casscf/v2rdm4/input.dat (hunk 2 above)
Comment (loriab) [reply tests/v2rdm_casscf/v2rdm4/input.dat:43]: yes, but it's 1 of 8 tests for a code where convergence is the big problem (maxiter 10k), so I'm not too downcast.

File: doc/sphinxman/document_tests.pl (hunk 1 above)
Comment (loriab) [reply doc/sphinxman/document_tests.pl:108]: especially perl you wrote!

**Pull request merged**